### ISSUE TITLE
Are you still taking patches for the 3.x branch?

### ISSUE THREAD

**Issue by adrienne** [oscarotero/Embed#461]
@oscarotero I have patched my local copy to fix the issue with the new Facebook opengraph stuff (based on your patch for the v4 branch here): https://github.com/oscarotero/Embed/blob/master/src/Adapters/Facebook/OEmbed.php#L24-L29 

And i suspect I'm not the only person who needs this! (We can't upgrade to v4 because of an incompatibility. I'm hoping we will soon, but in the meantime we are stuck on v3.)

**Comment by oscarotero** [comment]
Yes, sure. I'm not working actively for 3.x but accept patches from the community. So feel free to create a pull request.
Thanks!
